Uno.Gallery/Uno.Gallery.Shared/Views/Styles/NeumorphicStyles.xaml

```diff
@@ -0,0 +1,241 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:utu="using:Uno.Toolkit.UI"
+					xmlns:um="using:Uno.Material">
+
+	<ResourceDictionary.ThemeDictionaries>
+		<ResourceDictionary x:Key="Dark">
+			<!-- Shadow colors-->
+			<Color x:Key="LightShadowColor">#3C3846</Color>
+			<Color x:Key="ShadeShadowColor">#24222A</Color>
+		</ResourceDictionary>
+		<ResourceDictionary x:Key="Light">
+			<!-- Shadow colors-->
+			<Color x:Key="LightShadowColor">#FFFFFF</Color>
+			<Color x:Key="ShadeShadowColor">#AEAEC0</Color>
+		</ResourceDictionary>
+	</ResourceDictionary.ThemeDictionaries>
+
+	<!-- Shadow Collections -->
+		<utu:ShadowCollection x:Key="TextBoxShadows">
+			<utu:Shadow OffsetX="4"
+						OffsetY="4"
+						BlurRadius="4"
+						Color="{ThemeResource ShadeShadowColor}"
+						Opacity="0.4"
+						Spread="0"
+						IsInner="True" />
+			<utu:Shadow OffsetX="-4"
+						OffsetY="-4"
+						BlurRadius="4"
+						Color="{ThemeResource LightShadowColor}"
+						Opacity="0.4"
+						Spread="0"
+						IsInner="True" />
+		</utu:ShadowCollection>
+
+		<!-- Converters -->
+		<um:FromEmptyStringToValueConverter x:Key="EmptyToCompositeTransformTranslateY"
+											NotNullOrEmptyValue="8"
+											NullOrEmptyValue="0" />
+
+		<!-- TextBox Style -->
+		<Style x:Key="NeumorphicTextBoxStyle"
+			   TargetType="TextBox">
+			<Setter Property="Background"
+					Value="{ThemeResource SurfaceBrush}" />
+			<Setter Property="Foreground"
+					Value="{ThemeResource OnSurfaceBrush}" />
+			<Setter Property="PlaceholderForeground"
+					Value="{ThemeResource OnSurfaceVariantBrush}" />
+			<Setter Property="CornerRadius"
+					Value="8" />
+
+			<!-- Start: Body Large Typo -->
+			<Setter Property="FontFamily"
+					Value="{ThemeResource MaterialMediumFontFamily}" />
+			<Setter Property="FontWeight"
+					Value="{ThemeResource BodyLargeFontWeight}" />
+			<Setter Property="FontSize"
+					Value="{ThemeResource BodyLargeFontSize}" />
+			<Setter Property="CharacterSpacing"
+					Value="{ThemeResource BodyLargeCharacterSpacing}" />
+			<!-- End: Body Large Typo -->
+
+			<Setter Property="HorizontalContentAlignment"
+					Value="Left" />
+			<Setter Property="VerticalContentAlignment"
+					Value="Center" />
+
+			<Setter Property="Padding"
+					Value="16,16,14,16" />
+
+			<Setter Property="MinHeight"
+					Value="56" />
+
+			<!-- Workaround for WinUI issue: https://github.com/microsoft/microsoft-ui-xaml/issues/6388 -->
+			<Setter Property="um:ControlExtensions.Icon"
+					Value="{x:Null}" />
+
+			<Setter Property="Template">
+				<Setter.Value>
+					<ControlTemplate TargetType="TextBox">
```

Comment:
i made an updated style for textbox last time: https://github.com/unoplatform/uno.toolkit.ui/blob/dev/xygu/20230721/shadow-textbox/samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Content/Controls/ShadowContainerSamplePage.xaml#L138
why are we using the incorrect m3-ish style again as foundation?

Reply:
https://github.com/unoplatform/Uno.Themes/issues/974 is still a work in progress
Following the current Material TextBox behavior at the moment